packages/charts/src/LineChart/LineChart.tsx

```diff
@@ -200,8 +196,26 @@ const LineChart = React.forwardRef(function LineChart<X = unknown, Y = unknown>(
     marginBottom,
   } = dimensions;
 
-  const xDomain = getExtent(data, (d) => d[xKey], xDomainProp);
-  const yDomain = getExtent(data, (d) => d[yKey], yDomainProp);
+  let xGetter: (record: Record) => X;
+  if (typeof xValueSelector === 'string') {
```

Comment:
Would it be simpler to just support a function, or is it worth keeping support for strings?

Reply:
It doesn't add much complexity in code and perhaps someone may find it useful, so I'd keep it.